src/avatar.jsx

```diff
@@ -86,6 +83,10 @@ const Avatar = React.createClass({
     this.setState({muiTheme: newMuiTheme});
   },
 
+  getTheme() {
```

Comment:
> I went with this logic because it is used already in multiple components...

That's true, we should be consistent regarding this point, but that's not the case right now.
@alitaheri @newoga What do you think?